core/iwasm/include/wasm_c_api.h

```diff
@@ -200,6 +201,9 @@ wasm_config_set_mem_alloc_opt(wasm_config_t *, mem_alloc_type_t, MemAllocOption
 WASM_API_EXTERN own wasm_config_t*
 wasm_config_set_linux_perf_opt(wasm_config_t *, bool);
 
+WASM_API_EXTERN wasm_config_t*
+wasm_config_set_segue_flags(wasm_config_t *config, uint32_t segue_flags);
```

Comment:
do we have documentation about these flag bits?

Reply:
Not yet, how about adding comment here? For iwasm and wamrc, we had added comment in the command help info.

Reply:
> Not yet, how about adding comment here? For iwasm and wamrc, we had added comment in the command help info.

it sounds reasonable to me.

Reply:
Done.